api/inventory/v1/agents.proto

```diff
@@ -1439,7 +1439,7 @@ message ChangeNomadAgentParams {
 
 message AddValkeyExporterParams {
   // The pmm-agent identifier which runs this instance.
-  string pmm_agent_id = 1 [(validate.rules).string.uuid = true];
+  string pmm_agent_id = 1 [(validate.rules).string.min_len = 1];
```

Comment:
`pmm-server` is a valid agent ID and a valid node ID so previous validation would fail.